doc/source/admin/mq.md

```diff
@@ -1,3 +1,7 @@
+```eval_rst
+:orphan:
```

Comment:
Should we add this to the admin or special topics index page instead? (comment moved from https://github.com/galaxyproject/galaxy/pull/11101 , probably easier to do all reviewing here).

Reply:
Done.